release.config.js

```diff
@@ -7,27 +7,27 @@ module.exports = {
   branches: [
     {name: 'main', prerelease: 'main'},
     {name: 'master'},
-    {name: 'canary', prerelease: 'canary'}
+    {name: 'canary', prerelease: 'canary'},
   ],
   extends: [],
   plugins: [
     [
       '@semantic-release/commit-analyzer',
       {
         parserOpts: {
-          noteKeywords: ['BREAKING CHANGE', 'BREAKING CHANGES', 'BREAKING']
-        }
-      }
+          noteKeywords: ['BREAKING CHANGE', 'BREAKING CHANGES', 'BREAKING'],
+        },
+      },
     ],
     [
       '@semantic-release/release-notes-generator',
       {
         parserOpts: {
-          noteKeywords: ['BREAKING CHANGE', 'BREAKING CHANGES', 'BREAKING']
-        }
-      }
+          noteKeywords: ['BREAKING CHANGE', 'BREAKING CHANGES', 'BREAKING'],
```

Comment:
This is not extending like it should, will look into this. Thought this would have generated and now I took off the `CHANGELOG.md` 😭 

In the essence of time will put changelog back (and see what happens) but I have run out of (went way over) the time I had allotted for this today. 🤪